tomatoTomato/ContentView.swift

```diff
@@ -8,11 +8,41 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    @State private var isPresentingNewTaskView = false
+    
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        NavigationView {
+            Text("There will be a list of tasks here later")
+
```

Comment:
please remove this newline, it decreases readability

Reply:
done